config/v16/profile_schemas/Internal.json

```diff
@@ -292,7 +292,9 @@
         "ConnectorEvseIds": {
             "$comment": "Comma separated EVSEIDs for OCPP connectors starting with connector 1 in one string.",
             "type": "string",
-            "readOnly": false
+            "readOnly": false,
+            "minLength": 7,
+            "maxLength": 1000
```

Comment:
shoudn't this also be 37 ?

Reply:
its can be a comma seperated list, so Hubject specified this maxLength with 1000